lib/validators/doc-chart-validator.ts

```diff
@@ -53,17 +54,27 @@ export class DocChartValidator extends Validator {
 
         // Check whether the chart-properties control type property is applied to the doc-chart directive.
         for (const chartProperty of Object.values(this.chartPropertiesProperties(component))) {
-            this.validateChartProperty(component, chartProperty);
+            this.validateChartProperty(
+                component,
+                chartProperty as ComponentProperty<ComponentPropertyControlChartProperties>,
+            );
         }
     }
 
-    private validateChartProperty(component: Component, chartProperty: ComponentProperty) {
+    private validateChartProperty(
+        component: Component,
+        chartProperty: ComponentProperty<ComponentPropertyControlChartProperties>,
+    ) {
         if (!chartProperty.directiveKey) {
             this.error(
                 `Component "${component.name}" must configure "directiveKey" for the property with control type "chart"`,
             );
             return;
         }
+        if (!chartProperty.control.chartType) {
```

Comment:
Checking the existence can be removed again since it is already checked through the schema. Same for the test.